internal/mapping.go

```diff
@@ -149,25 +149,43 @@ func ToSpannerForeignKey(conv *Conv, srcFkName string) string {
 // a) CASCADE/NO ACTION -> mapped to the same as source action
 // b) all others -> NO ACTION (default)
 //
-// Since only MySQL and PostgreSQL have this functionality
-// as of yet, for other sources OnDelete fields are
-// kept empty i.e. "" is mapped to ""
-//
 // For all source actions converted to a different action,
 // an issue is appended to it's TableLevelIssues to
 // generate a warning message for the user
+//
+// Since only MySQL and PostgreSQL have this functionality
+// as of yet, for other sources OnDelete fields are
+// kept empty i.e. "" is mapped to "" and an issue is appended
+// to generate warning message (prints only once for each table with FK Actions)
 func ToSpannerOnDelete(conv *Conv, srcTableId string, srcDeleteRule string) string {
 	srcDeleteRule = strings.ToUpper(srcDeleteRule)
-	if srcDeleteRule == constants.NO_ACTION || srcDeleteRule == constants.CASCADE || srcDeleteRule == "" {
+	if srcDeleteRule == constants.NO_ACTION || srcDeleteRule == constants.CASCADE {
```

Comment:
nit: Coment related to a previous change, we should actually keep the constant variable name ot FK_NO_ACTION , FK_CASCADE or FOREIGN_KEY_NO_ACTION (thats upto you) but itd be nice if we can mkeep the naming specific